packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -174,8 +174,11 @@ export default class extends Component {
       })
     }
 
-    const smartctlHealth = await getSmartctlHealth(host)
-    const isSmartctlHealthEnabled = smartctlHealth !== null
+    let smartctlHealth
+    try {
+      smartctlHealth = await getSmartctlHealth(host)
+    } catch (error) {}
+    const isSmartctlHealthEnabled = smartctlHealth != null
     const smartctlUnhealthyDevices = isSmartctlHealthEnabled
       ? Object.keys(smartctlHealth).filter(deviceName => smartctlHealth[deviceName] !== 'PASSED')
       : undefined
```

Comment:
I think there is a specific error if plugin is missing , something with a code like `PLUGIN_NOT_INSTALLED, this way we won't eat any other error

```suggestion
    let smartctlHealth
    try {
      const smartctlHealth = await getSmartctlHealth(host)
      const smartctlUnhealthyDevices =  smartctlHealth !== null ? Object.keys(smartctlHealth).filter(deviceName => smartctlHealth[deviceName] !== 'PASSED')
    } catch (error) {
        if(error.code !=='THE EXPECTED_CODE'){
              throw error
        }
    }

```